### DISCUSSION TITLE
Add more inverters into the system

### DISCUSSION THREAD

**Discussion by ClemensReichel** [pvlib/pvlib-python#2034] [Q&A] [answered]
How would you add more inverters into the system? With "modules_per_string" and "strings_per_inverter" you can layout the PV modules. But how would you do the same for the inverters?

**Comment by kandersolar** [comment, marked as answer]
Hi @ClemensReichel, pvlib's `PVSystem` and `ModelChain` support only a single inverter.  In general, simulating multiple inverters is done by defining a `PVSystem`/`ModelChain` instance for each inverter.  Or in the special case where all aspects of the inverters and their arrays are identical, you just perform one simulation and multiply by the number of inverters.

See also: https://pvlib-python.readthedocs.io/en/stable/user_guide/faq.html#how-do-i-model-a-system-with-multiple-inverters
